python/strings_udf/strings_udf/__init__.py

```diff
@@ -14,8 +14,48 @@
 
 logger = get_logger()
 
-# tracks the version of CUDA used to build the c++ and PTX components
-STRINGS_UDF_PTX_VERSION = (11, 8)
+
+def get_cuda_version_from_ptx_file(path):
```

Comment:
Should this be private?
```suggestion
def _get_cuda_version_from_ptx_file(path):
```